ui/app/store/actions.js

```diff
@@ -2299,6 +2299,16 @@ export function setMkrMigrationReminderTimestamp (timestamp) {
   }
 }
 
+export function setConnectedStatusPopoverHasBeenShown () {
+  return (dispatch) => {
+    background.setConnectedStatusPopoverHasBeenShown((err) => {
+      if (err) {
+        return dispatch(displayWarning(err.message))
```

Comment:
This is in keeping with the pattern used for many other actions, but practically this `displayWarning` method is both ineffectual and actively confusing. It shows this global warning all over the place (settings, account details, confirm transaction screen) regardless of applicability, and if it did happen in this particular instance it'd be hidden from the user (it's not shown on the Home screen).

At the very least we should be capturing this with Sentry. Throwing it here should result in it being captured, since it'd be uncaught. Or you could capture it explicitly.

Reply:
Done in 593fdffa2